src/Cron_Command.php

```diff
@@ -295,8 +342,14 @@ public function run_now( $args ) {
 		if ( empty( $result ) ) {
 			EE::error( 'No such cron with id ' . $args[0] );
 		}
-		$container = $this->site_php_container( $result['site_url'] );
-		$command   = $result['command'];
+
+		$container = $this->site_php_container( $result->site_url );
+		$command   = $result->command;
+
+		if ( 'host' === $result->site_url ) {
+			EE::exec( $command, true, true );
```

Comment:
Should `return` after executing this. You don't want to execute the `docker exec` command below in case of `host`.